File: src/fractal/demon/basic/baseline.py
# """
#     Benchmark.MinMaxTS(;depth, τ=0.) <: Benchmark.Player

# Minmax baseline, which relies on [`MinMax.Player`](@ref).
# """
# @kwdef struct MinMaxTS <: Player
#   depth :: Int
#   amplify_rewards :: Bool
#   τ :: Float64 = 0.
# end

# name(p::MinMaxTS) = "MinMax (depth $(p.depth))"

# def instantiate(p::MinMaxTS, ::AbstractGameSpec, nn)
#   return MinMax.Player(
#     depth=p.depth, amplify_rewards=p.amplify_rewards, τ=p.τ)

File: src/fractal/demon/naivety/baseline.rs
//! Benchmark.MinMaxTS(;depth, τ=0.) <: Benchmark.Player
// # Minmax baseline, which relies on <`MinMax.Player`>(@ref).
// # \\\!

pub struct Minimax {
    depth: i32,
    amplify_rewards: bool,
    tau: f64,
}

impl Display {
    // # name(p::MinMaxTS) = "MinMax (depth $(p.depth))"
}


fn new(minimax: Minimax, nature: Nature, brain: Brain) {
    MinMax.Player { depth: minimax.depth, amplify_rewards: minimax.amplify_rewards, tau: minimax.τ }
}
